src/features/CourtCaseDetails/Sidebar/PncDetails/PncCourtCaseAccordion.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react"
+import type { PncCourtCase } from "@moj-bichard7-developers/bichard7-next-core/core/types/PncQueryResult"
+import {
+  CCR,
+  CourtCaseHeaderContainer,
+  CrimeOffenceReference,
+  CourtCase,
+  Offence,
+  CourtCaseHeader,
+  ChevronContainer
+} from "./PncDetails.styles"
+import PncOffenceDetails from "./PncOffenceDetails"
+import Disposal from "./Disposal"
+
+interface PncCourtCaseAccordionProps {
+  index: number
+  pncCourtCase: PncCourtCase
+}
+
+const PncCourtCaseAccordion = ({
+  index,
+  pncCourtCase: { courtCaseReference, crimeOffenceReference, offences }
+}: PncCourtCaseAccordionProps) => {
+  const [open, setOpen] = useState<boolean>(index === 0 ? true : false)
+  const handleClick = () => {
+    setOpen(!open)
+  }
```

Comment:
```suggestion
  const handleClick = () => setOpen((previousState) => !previousState)

```

Reply:
Also, why did we remove the `toggle` word? In `src/components/Accordion.tsx` it's used. I'd like to keep things consistent with the Accordion.